src/modules/site-v2/base/views/strains.py

```diff
@@ -205,7 +293,23 @@ def order_confirmation(invoice_hash):
                           for x in order_obj['items'].split("\n")}
     if order_obj is None:
       abort(404)
-    title = f"Invoice {order_obj['invoice_hash']}"
+    title = "Order Confirmation"
+    invoice = f"Invoice {order_obj['invoice_hash']}"
     return render_template('order/order_confirm.html', **locals())
   else:
     abort(404)
+    
+
+
+@strains_bp.route('/submit')
+@cache.memoize(60*60)
+def strains_submission_page():
+  """
+      Google form for submitting strains
+  """
+  title = "Strain Submission"
+  # TODO: Move this to configurable location
+  #STRAIN_SUBMISSION_FORM = '1w0VjB3jvAZmQlDbxoTx_SKkRo2uJ6TcjjX-emaQnHlQ'
+  #strain_submission_url = f'https://docs.google.com/forms/d/{STRAIN_SUBMISSION_FORM}/viewform?embedded=true'
+  strain_submission_url = "https://docs.google.com/forms/d/1w0VjB3jvAZmQlDbxoTx_SKkRo2uJ6TcjjX-emaQnHlQ/viewform?embedded=true"

```

Comment:
Since this URL will change based on what environment the application is running, it should be promoted to a env/development/global.env variable. (and it should be propagated to other environments, eg: main).